rj_gameplay/rj_gameplay/tactic/wall_tactic.py

```diff
@@ -20,108 +20,72 @@
 class wall_cost(role.CostFn):
     """Cost function for role request.
     """
+    def __init__(self, wall_pt: np.ndarray=None):
+        self.wall_pt = wall_pt
+
     def __call__(
         self,
         robot: rc.Robot,
         prev_result: Optional["RoleResult"],
         world_state: rc.WorldState
     ) -> float:
 
-        # TODO: make closest robots form wall, rather than setting on init
-        # aka actually use this method
-        # can do this by creating mult cost instances, adding attr like "wall_pt" to this class, 
-        # and giving each roleRequest the correct cost in tactic below
-        return 0.0
+        if robot is None or self.wall_pt is None:
+            return 0
+        return np.linalg.norm(robot.pose[0:2]-self.wall_pt)
```

Comment:
Nit: units for cost functions should be seconds, not distance. Divide by max robot velocity from global constants.

Reply:
Good to know